src/main/java/vectorwing/farmersdelight/integration/crafttweaker/MCToolIngredient.java

```diff
@@ -1,97 +0,0 @@
-//package vectorwing.farmersdelight.integration.crafttweaker;
```

Comment:
This question is a bit weird, since I wasn't involved in writing this class:
I noticed that `MCToolIngredient.java` was just removed, not moved to another package. Is it no longer needed for writing tool ingredients?